impeller/entity/shaders/filters/gaussian.frag

```diff
@@ -18,7 +18,7 @@ struct KernelSample {
 
 uniform KernelSamples {
   int sample_count;
-  KernelSample samples[48];
+  KernelSample samples[50];
```

Comment:
aka `kMaxKernelSize`?

Reply:
Oh wait, is that constant available from the frag? Is this something the frag should define and export in its generated header so that the C++ code can refer to it?

Reply:
Nope, the impellerc compiler doesn't export those.  There already was a `kSampleCount` that didn't get carried over to the header.